pkg/build/docker_generic.go

```diff
@@ -38,20 +38,17 @@ func (b *DockerGenericBuilder) Build(ctx context.Context, in *api.BuildInput, ow
 	cliopts := []client.Opt{client.FromEnv, client.WithAPIVersionNegotiation()}
 
 	var (
-		id       = in.BuildID
 		basesrc  = in.BaseSrcPath
 		cli, err = client.NewClientWithOpts(cliopts...)
 	)
 	if err != nil {
 		return nil, err
 	}
 
-	ow = ow.With("build_id", id)
 	ctx, cancel := context.WithTimeout(ctx, 5*time.Minute)
 	defer cancel()
 
 	opts := types.ImageBuildOptions{
-		Tags:        []string{id, in.BuildID},
```

Comment:
Previously this was creating the default tag, but it creates `latest` tag, and we don't really want that.